### ISSUE TITLE
Question in the sampling stage

### ISSUE THREAD

**Issue by medcx** [MICV-yonsei/CT2MRI#4]
The style key of the validation set should be unknown, but you generate it directly during data preprocessing and use it directly in the sampling stage. Is this reasonable?

**Comment by CKBling** [comment]
Hello,

What you mentioned refers to the "Ours_best" option in the paper, and you're right—it isn't a true validation (test). The code is set up so that you can test using other options, like "Ours_avg," which only uses the style key from the training set.

**Comment by medcx** [comment]
Another question is that the article says that BraTs processes to 176*176*160, but I found that the original data is 240*240*155. Your processing method does not mention how to expand 155 to 160. Does it use zero padding?
